src/components/HourChart/HourChart.jsx

```diff
@@ -364,7 +371,13 @@ export default class HourChart extends PureComponent {
    * @return {React.Component} The rendered container
    */
   render() {
-    const { width, id, height } = this.props;
+    const { dataByHour } = this.prepareData(this.props);
+    const { width, id, height, threshold } = this.props;
+    let { yExtent } = this.props;
+
+    if (!yExtent) {
+      yExtent = [0, threshold];
+    }
 
```

Comment:
Not clear why this is happening here, but it could probably happen in BarChart if it is necessary.